### ISSUE TITLE
 Detections are extremely wrong.

### ISSUE THREAD

**Issue by devacharan** [matterport/Mask_RCNN#2958]
Hello everyone,

I was enthusiastic about applying this project to a new dataset, but I have been facing challenges in obtaining promising results. Despite encountering impressive examples of successful outcomes, I suspect that I may have overlooked a fundamental aspect in my setup.

My aim to detect all axis and labels in a plot. 
My dataset consists of  images of plots, with 6 classes: axis and labels
Images:  training: 250 and, I checked my annotations using the inspect_model_data.ipynb as a guide, everything seems fine.

Detection test using the inspect_model.ipynb as a guide ends up with something looking pretty similar to this:

![image](https://github.com/matterport/Mask_RCNN/assets/75236563/7e0d6a8f-a139-4b42-b342-b3950143f167)

I am keep getting the same results irrespective different learning rates, epochs etc...
I am using tensor flow version-2.5

my questions are,
Is my dataset simply too small for the complexity a Resnet101 backbone?
or I'm screwing up a fundamental aspect of my config?
 
So, any initial thoughts on where I'm going wrong?

**Comment by avinash-218** [comment]
i am facing similar issue. the training and validation loss is around 0.3. but the mAp is0. and my detections are similar to the image of @devacharan

**Comment by avinash-218** [comment]
I downgraded my tensorflow version from 2.7 to 2.5 - worked !!!

Now the mAp, mAR, F1 score are around 60%.
